src/MonsterImage.tsx

```diff
@@ -48,6 +49,9 @@ export function MonsterImage({
         height={96 * scale}
         onLoad={setLoaded}
         onError={setErrored}
+        onClick={() => {
+          setShiny(!shiny);
+        }}
```

Comment:
This works well, but it has both accessibility and discoverability problems. People will probably only find out about this on accident. It would be a shame for this lovely new feature to go unnoticed.

This could merge as-is since it's not a core feature, but I'm trying to think of a way to make sure users can tell you can view shiny forms (especially since this code doesn't KNOW if we have a shiny form available right here...)

In an ideal world, I feel like there would be some kind of way to cycle through all the alternate forms of a Pokemon sprite here, but that's way more work and less important than shinies.

My initial temptation was a hover cursor or hover color, but that won't help on mobile. Maybe a little button with a sparkle in the top right corner of the card could work for toggling shiny display. I could definitely make an icon given some time.